contracts/pallet-ibc/src/light_clients.rs

```diff
@@ -334,69 +328,19 @@ pub enum AnyClientMessage {
 }
 
 impl AnyClientMessage {
-	pub fn maybe_header_height(&self) -> Option<Height> {
-		match self {
-			Self::Tendermint(inner) => match inner {
-				ics07_tendermint::client_message::ClientMessage::Header(h) => Some(h.height()),
-				ics07_tendermint::client_message::ClientMessage::Misbehaviour(_) => None,
-			},
-			Self::Beefy(inner) => match inner {
-				ics11_beefy::client_message::ClientMessage::Header(_) =>
-					unimplemented!("beefy header height"),
-				ics11_beefy::client_message::ClientMessage::Misbehaviour(_) => None,
-			},
-			Self::Grandpa(inner) => match inner {
-				ics10_grandpa::client_message::ClientMessage::Header(h) => Some(h.height()),
-				ics10_grandpa::client_message::ClientMessage::Misbehaviour(_) => None,
-			},
-			Self::Wasm(inner) => match inner {
```

Comment:
@misko9 just to help speed up the review on this pr a bit, but was it intended that these misbehaviour client messages were removed on this PR? there are some other bits around the PR which have logic removed/set to "unimplemented" as well, just trying to understand!